File: rest.vlabs.com.ua/src/main/java/vlabs/model/HelpClip.java
package vlabs.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@Entity
@Table(name="helpclips")
@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
public class HelpClip {

    @Id
    @Column(name = "id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "title", length = 255)
    private String title;

    @Column(name = "alias", length = 255)
    private String alias;

    @Column(name = "path", length = 2048)
    private String path;

    @Column(name = "shortdesc", length = 1024)
    private String shortdesc;

    @Column(name = "description", length = 2048)
    private String description;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getAlias() {
        return alias;
    }

    public void setAlias(String alias) {
        this.alias = alias;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    public String getShortdesc() {
        return shortdesc;
    }

    public void setShortdesc(String shortdesc) {
        this.shortdesc = shortdesc;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }


}

File: vlabs1/ff.vlabs.com.ua/src/app/model/helpclip/helpclip.ts
export class HelpClip {
    id: number;
    title: string;
    alias: string;
    path: string;
    shortdesc: string;
    description: string;

    constructor() {
        this.id = undefined;
        this.title = undefined;
        this.alias = undefined;
        this.path = undefined;
        this.shortdesc = undefined;
        this.description = undefined;
    }

    map(anyHelpClipItem: any): HelpClip {
        let helpClip: HelpClip = new HelpClip();
        Object.keys(anyHelpClipItem).forEach(key => {
            if (helpClip.hasOwnProperty(key)) {
                helpClip[key] = anyHelpClipItem[key];
            }
        });
        return helpClip;
    }
}